src/index.html

```diff
@@ -1,36 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-     <script src="https://kit.fontawesome.com/f5f70a395d.js"
-     crossorigin="anonymous"></script>
-     
-    <title>To do List App</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Comic+Neue" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/f5f70a395d.js"
+    crossorigin="anonymous"></script>
+    <title> To do List</title>
 </head>
+
 <body>
     <div class="all-container">
-       <div class="container-main">
-         <div class="header-list">
-            <h1>To-do Task</h1>
-            <i class="fa-solid fa-rotate pointer" id="refresh"></i>
-         </div>
-
-         <div class="container-form">
-        <form action="" id="new-task" class="task-container row">
-           <input type="text" class="input" placeholder="Add to your list..." id="task-description">
-           <button type="submit" class="pointer"><i class="fa fa-reply"></i></button>
-        </form>
-
-        <ul class="todo-list" id="todo-list">
-        </ul>
-
-        <div class="cleartodo">
-            <a href="" class="clear-completed-task">Clear all completed</a>
+        <div class="container-main">
+          <div class="header-list">
+             <h1>To-do Task</h1>
+             <i class="fa-solid fa-rotate pointer" id="refresh"></i>
+          </div>
+          <div>
+            <ul class="todo-list" id="todo-list">
+            </ul>
         </div>
-         </div>
-       </div>
-
-    </div>
+          <div class="container-form">
+                
+             <div class="wrapper">
+              <form action="" id="new-task" class="task-container row">
+                <input type="text" class="input" placeholder="Add to your list..." id="task-description">
+                <!-- <input type="text" name="title" id="title" placeholder="Title" class="input_title"> <br>
+                <input type="text" name="Author" placeholder="Author" id="author" class="input_author"> <br> -->
```

Comment:
Please don't leave any uncommented code in your codebase. In this way, you can keep your codebase neat, clean, and professional. 👨‍💼